### DISCUSSION TITLE
How to get badges?

### DISCUSSION THREAD

**Discussion by noActive** [community/community#19820] [Other Feature Feedback, Questions, & Ideas]
Hi
I have seen badges/achievements on the GITHUB profile in a large number of people once, I would like to know if there is any possibility of getting it? I am very curious, please let me know quickly
Have a nice day <3

**Comment by julianalsemmani** [comment]
Hey!

By hovering over the badge, it will give you a small pop-up with how it was obtained.

For example:
```
Galaxy Brain
@username answered discussions.
```

```
Pull Shark
@username opened pull requests that have been merged.
```

So, that's how you get the different badges.

See more here:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by tvnr** [comment]
Pull Shark Badge by merging PRs. Own repository works. Galaxy brain badge by answering two discussions
You also have one for making a PR and getting it accepted without it getting reviewed.

You can see more information at the following link:
https://github.com/Schweinepriester/github-profile-achievements

**Reply by tvnr** [reply]
I will be glad if you confirm my answer :))

**Comment by Zi-Tao** [comment]
@nayyyan https://github.com/Schweinepriester/github-profile-achievements/blob/main/README.md
<img width="660" alt="截圖 2022-07-01 下午10 00 46" src="https://user-images.githubusercontent.com/80264433/176909556-6df10f61-1c0d-460e-8c49-77dadae044e3.png">

**Comment by noActive** [comment]
Welcome back
Thank you for answering my question!

**Comment by AF2828** [comment]
Well, the easiest ones are to create pull requests and merges. Maybe start there 😉

Than you have other ones like becoming a sponsor or donate programs to archives or contributing to git hub which I am not sure I will ever get
